static/css/photon-site/footer.less

```diff
@@ -0,0 +1,194 @@
+.Footer {
```

Comment:
This was also based on the frontend css.  Stylus is a little different to less but I think the resulting css should be the same if I rewrote it correctly.